projects/storefrontapp-e2e-cypress/cypress/integration/vendor/cds/profile-tag.e2e-spec.ts

```diff
@@ -1,13 +1,13 @@
 import * as anonymousConsents from '../../../helpers/anonymous-consents';
```

Comment:
Other than the minor changes/feedback below

The test is fluid and nice :) passing 100%